features/support/env.rb

```diff
@@ -9,27 +9,62 @@
 require 'capybara'
 require 'capybara/cucumber'
 require 'selenium/webdriver'
-require 'chromedriver-helper'
+require 'webdrivers'
 require 'cucumber/rails'
 require 'cucumber/rspec/doubles'
 require 'site_prism'
+require_relative '../page_objects/base_page'
 require 'sidekiq/testing'
+require_relative '../page_objects/base_page'
 require_relative '../../spec/vcr_helper'
 require_relative '../../spec/support/factory_helpers'
 
+# disable forgery protection in feature tests so as not to obscure
+# loss of signed in user
+ActionController::Base.allow_forgery_protection = false
```

Comment:
this is default set in `test.rb` either enable (set true) here or remove/comment out